tensorflow/compiler/tf2tensorrt/tensorrt_test.cc

```diff
@@ -22,29 +22,19 @@ limitations under the License.
 #include "third_party/gpus/cuda/include/cuda.h"
 #include "third_party/gpus/cuda/include/cuda_runtime_api.h"
 #include "third_party/tensorrt/NvInfer.h"
+#include "tensorflow/compiler/tf2tensorrt/utils/trt_logger.h"
 
-namespace tensorflow {
-namespace {
+//TODO(mconley): Should the below macro and the one in convert/utils.h be moved to an upper utils directory?
```

Comment:
Good point, there is a similar macro in tf2tensorrt/utils/trt_logger.cc and one in trt_tensor_proxy.h.
We can put it to tf2tensorrt/common/utils.h.

Reply:
Sounds good, I've moved the macro there and adjusted the imports accordingly